modules/security-context-constraints-about.adoc

```diff
@@ -366,16 +366,15 @@ pod to fail.
 [id="scc-prioritization_{context}"]
 == Security context constraints prioritization
 
-Security context constraints (SCCs) have a priority field that affects the ordering when attempting to
-validate a request by the admission controller. A higher priority
-SCC is moved to the front of the set when sorting. When the complete set
-of available SCCs are determined they are ordered by:
+Security context constraints (SCCs) have a priority field that affects the ordering when attempting to validate a request by the admission controller. A priority value of `0` is the lowest possible priority. A nil priority is considered a `0`, or lowest, priority. Higher priority SCCs are moved to the front of the set when sorting.
 
-. Highest priority first, nil is considered a 0 priority
-. If priorities are equal, the SCCs will be sorted from most restrictive to least restrictive
-. If both priorities and restrictions are equal the SCCs will be sorted by name
+When the complete set of available SCCs is determined, the SCCs are ordered in the following manner:
+
+. The highest priority SCCs are ordered first.
+. If the priorities are equal, the SCCs are sorted from most restrictive to least restrictive.
+. If both the priorities and restrictions are equal, the SCCs are sorted by name.
 
 By default, the `anyuid` SCC granted to cluster administrators is given priority
 in their SCC set. This allows cluster administrators to run pods as any
 user by without specifying a `RunAsUser` on the pod's `SecurityContext`. The
-administrator may still specify a `RunAsUser` if they wish.
+administrator can still specify a `RunAsUser` if they want.
```

Comment:
Since we're fixing vocabulary here, I am actually quite puzzled about what this paragraph wants to say.

```
This allows cluster administrators to run pods as any
user by without specifying a `RunAsUser` on the pod's `SecurityContext`
```
It allows them to run as any user exactly by specifying the `RunAsUser`.

But then the following sentence does not make any sense:
```
The administrator can still specify a `RunAsUser` if they want.
```
?

Reply:
@stlaz I'm honestly not sure since I didn't write the original content. 

I checked and this content was brought forward from 3.x: https://docs.openshift.com/container-platform/3.11/architecture/additional_concepts/authorization.html#scc-prioritization

It actually dates back to 3.1, and was added with this PR: #1078. I don't see anything from the discussion in that PR though that helps explain what the intended meaning was here.

Let me know if you have any ideas on how we should change it. Is just the first sentence really the only relevant one to priority, and we should remove the others if they're not accurate?